### PR TITLE
Improve 'README.md' file

### PR DESCRIPTION
**Pull request by FaaPz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,78 @@
...
-Or you can add it directly in your composer.json file:
+Or add it to `composer.json` manually:
+
 ```json
-{
-	"require": {
-    	    "sjongejan/laravel-bag": "dev-master"
-	}
+"require": {
+    "sjongejan/laravel-bag": "dev-master"
```
Comment (sjongejan) [inline README.md]: Target to version 0.1.0?

File: README.md [hunk 2]
```diff
@@ -1,63 +1,78 @@
...
 }
 ```
 
-### Register service provider and facade
-Register the provider directly in your app configuration file `config/app.php`:
+Laravel 5.5+ will use its [auto-discovery](https://laravel.com/docs/5.5/packages#package-discovery) functionality.
```
Comment (sjongejan) [inline README.md]: This is not implemented in the composer.json? 🤔

Comment (FaaPz) [reply README.md]: It is! See commit 258858ad77838225350347234fd19647ee2e5560. 🙌

You've approved it two hours ago... 🙊

Comment (sjongejan) [reply README.md]: Hmmm, I am not awake.

**Pull request merged**